notebooks/data_processing.ipynb

```diff
@@ -0,0 +1,795 @@
+{
```

Comment:
<p>We should try to avoid <code>save_path</code> which are not writable. For example, if I run this notebook, I will not be able to save data at this path. If this is a temporary file, you can save it at <code>/tmp</code>  directory otherwise you can use our shared <code>/opt/kdd24</code> path. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>